src/app/core/services/authenticate.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { ApiService, AuthToken, UserDTO } from '../../../backend-client';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthenticateService {
```

Comment:
I believe this is also a typo. Should be `AuthenticationService` or `AuthService`. 
Authenticate (verb) => to log in
Authenticated (verb) => Already logged in
Authentication (noun) => About logging in
Don't worry much, you'll eventually improve !